src/main.py

```diff
@@ -10,12 +10,12 @@
          content_security_policy_nonce_in=['script-src'])
 logging.basicConfig(level=logging.DEBUG)
 
-
 @app.route('/')
 @app.route('/<lang>/')
 @validate
 def index(lang):
-    return render_template('%s/splash.html' % lang)
+    supported_languages = SUPPORTED_YEARS.get('2019')
+    return render_template('%s/splash.html' % lang, supported_languages=supported_languages)
```

Comment:
`supported_languages` must be passed into all templates that include the language switcher.